src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -916,24 +917,24 @@
     </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Condition="'$(TargetsWindows)' == 'true' and '$(IsUAPAssembly)' != 'true'" Include="Microsoft.Win32.Registry" Version="$(MicrosoftWin32RegistryVersion)" />
+    <PackageReference Condition="'$(TargetsWindows)' == 'true' and '$(IsUAPAssembly)' != 'true' and ('$(TargetGroup)' == 'netstandard' or ('$(TargetGroup)' == 'netcoreapp' and '$(_TargetFrameworkVersionWithoutV)' &lt;= '6.0'))" Include="Microsoft.Win32.Registry" Version="$(MicrosoftWin32RegistryVersion)" />
     <PackageReference Include="System.Configuration.ConfigurationManager" Version="$(SystemConfigurationConfigurationManagerVersion)" />
     <PackageReference Include="System.Security.Permissions" Version="$(SystemSecurityPermissionsVersion)" />
-    <PackageReference Include="System.Security.Principal.Windows" Version="$(SystemSecurityPrincipalWindowsVersion)" />
-    <PackageReference Include="System.Text.Encoding.CodePages" Version="$(SystemTextEncodingCodePagesVersion)" />
-    <PackageReference Include="System.Text.Encodings.Web" Version="$(SystemTextEncodingsWebVersion)" />
+    <PackageReference Condition="'$(TargetGroup)' == 'netstandard' or ('$(TargetGroup)' == 'netcoreapp' and '$(_TargetFrameworkVersionWithoutV)' &lt;= '6.0')" Include="System.Security.Principal.Windows" Version="$(SystemSecurityPrincipalWindowsVersion)" />
+    <PackageReference Condition="'$(TargetGroup)' == 'netstandard' or ('$(TargetGroup)' == 'netcoreapp' and '$(_TargetFrameworkVersionWithoutV)' &lt;= '6.0')" Include="System.Text.Encoding.CodePages" Version="$(SystemTextEncodingCodePagesVersion)" />
+    <PackageReference Condition="'$(TargetGroup)' == 'netstandard' or ('$(TargetGroup)' == 'netcoreapp' and '$(_TargetFrameworkVersionWithoutV)' &lt;= '6.0')" Include="System.Text.Encodings.Web" Version="$(SystemTextEncodingsWebVersion)" />
     <PackageReference Include="Microsoft.Data.SqlClient.SNI.runtime" Version="$(MicrosoftDataSqlClientSNIRuntimeVersion)" />
-    <PackageReference Condition="$(TargetGroup) == 'netcoreapp' " Include="System.Diagnostics.DiagnosticSource" Version="$(SystemDiagnosticsDiagnosticSourceVersion)" />
+    <PackageReference Condition="'$(TargetGroup)' == 'netstandard' or ('$(TargetGroup)' == 'netcoreapp' and '$(_TargetFrameworkVersionWithoutV)' &lt;= '6.0')" Include="System.Diagnostics.DiagnosticSource" Version="$(SystemDiagnosticsDiagnosticSourceVersion)" />
     <PackageReference Include="Azure.Identity" Version="$(AzureIdentityVersion)" />
```

Comment:
This will need to be made to not reference the nuget package versions of the assemblies in the normal runtime and ref pack files as well when targeting .NET 6+ (will require PR to their repo).